### ISSUE TITLE
Camp release tarball w/ submodules

### ISSUE THREAD

**Issue by cyrush** [LLNL/camp#111]
can you provide one? :-)

**Comment by trws** [comment]
Yup, I will figure this out for subsequent releases.  Do you need one for the most recent release as well?

**Comment by cyrush** [comment]
If possible that would be nice, I would like to grab (vs git clones) in some of scripts.
